bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -93,6 +93,24 @@ spec:
           value: {{ .Values.cluster.datacenter }}
         - name: CASSANDRA_ENDPOINT_SNITCH
           value: {{ .Values.cluster.endpointSnitch }}
+{{- if .Values.tlsEncryptionSecretName }}
+        - name: CASSANDRA_INTERNODE_ENCRYPTION
+          value: {{ .Values.cluster.internodeEncryption }}
+        - name: CASSANDRA_CLIENT_ENCRYPTION
+          value: {{ .Values.cluster.clientEncryption }}
+        - name: CASSANDRA_TRUSTSTORE_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.tlsEncryptionSecretName }}
```

Comment:
Should this secret be 'auto-generated' when not specified explicitly?

Reply:
I don't think you can autogenerate the keystore and truststore files

Reply:
We could, but that would require creating a job that auto-generates it (with the proper RBAC permissions). In this first version, the user will need to provide